rfc/eol_kubernetes_v1.md

```diff
@@ -0,0 +1,109 @@
+
+| Spinnaker Version | Release Window Opens | Milestones |
+|-------------------|----------------------|------------|
+| 1.18              | 2020-01-06           | Gain consensus on this RFC. Highlight in 1.18 curated changelog that V1 will be removed in 1.21.
+| 1.19              | 2020-03-02           | Publish a post to the Spinnaker Community Blog explaining why we are removing V1 and how to migrate to V2. Add a feature flag that must be enabled before deploying to V1 accounts. Add appropriate warnings to Halyard, Clouddriver, and Deck. Highlight in 1.19 curated changelog that V1 will be removed in 1.21.
+| 1.20              | 2020-04-27           | Complete audit of community docs and blog posts for V1-related content to update. Highlight in 1.20 curated changelog that V1 will be removed in 1.21.
+| 1.21              | 2020-06-22           | Remove V1-only code from all Spinnaker microservices.
+
+## Design
+
+- [x] Determine if there are user stories addressed by the V1 provider but not the V2 provider that we should consider adding to V2 before removing V1.
+  - [x] Identify and interview existing V1 users in order to learn why they are still using the V1 provider (Thank you to everyone who took the time to speak with us!).
+  - [x] Identify and interview people currently evaluating Spinnaker in order to learn if the level of abstraction exposed by V1’s wizards would have value as an onboarding tool in V2
+  (Short answer: most users think that manifest composition is now out of scope for Spinnaker because there is now such a rich ecosystem of Kubernetes-native tooling for manifest templating).
+
+- [x] Change documentation to encourage new users to adopt only the V2 provider ([spinnaker.github.io/pull/1586](https://github.com/spinnaker/spinnaker.github.io/pull/1586)).
+  - [x] Change the docs to explicitly endorse V2 as the recommended provider ([spinnaker.github.io/pull/1586](https://github.com/spinnaker/spinnaker.github.io/pull/1586)).
+  - [x] Remove the V1 provider codelab ([spinnaker.github.io/pull/1586](https://github.com/spinnaker/spinnaker.github.io/pull/1586)). Add a redirect from the V1 codelab to the V2 codelab ([spinnaker.github.io/pull/1649](https://github.com/spinnaker/spinnaker.github.io/pull/1649)).
+  - [x] Remove inbound links to V1 provider setup ([spinnaker.github.io/pull/1649](https://github.com/spinnaker/spinnaker.github.io/pull/1649)).
+  
+- [] Publish a blog post on the Spinnaker Community Blog. This post will answer the following questions:
+  - [] Why is Spinnaker removing the V1 provider?
+  - [] When is Spinnaker removing the V1 provider? What is the latest supported release I will be able to use with the V1 provider enabled?
+  - [] How can I successfully execute a zero-downtime migration from the V1 provider to the V2 provider? What are some first-hand lessons from teams who have done this?
+
+- [] Audit existing Spinnaker content to update.
```

Comment:
Feel free to leverage any of our techincal writers to help in this effort! cc @brian-armory.

Reply:
Thank you! :pray: :hugs: